### PR TITLE
24289 - style: fix flexgrid to wrap on small sized screens and smaller

### PR DESCRIPTION
**Pull request by hfekete**
*Issue:24289 *https://github.com/bcgov/entity/issues/24289 

*Description of changes:*
 fix style for flexgrid to wrap on small sized screens and smaller

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-71-6sv5hfgs.web.app

File: src/components/bcros/businessDetails/index.vue [hunk 1]
```diff
@@ -94,9 +94,13 @@ watch(() => route.params.identifier as string, loadComponentData, { immediate: t
 </script>
 
 <template>
-  <div id="bcros-business-details" class="bg-white h-[150px]" data-cy="business-details">
-    <div class="flex pt-5 text-bcGovGray-900 app-inner-container">
-      <div class="grow" data-cy="business-details-name">
+  <div
+    id="bcros-business-details"
```
Comment (patrickpeinanw) [inline src/components/bcros/businessDetails/index.vue:98]: In the temp url, there are some extra space at the top of the tombstone. I guess it is caused by this 'pt-7'.

File: src/components/bcros/header/Index.vue [hunk 2]
```diff
@@ -4,10 +4,7 @@
     class="bg-bcGovColor-darkBlue border-b-2 border-yellow-400"
     data-cy="bcros-header"
   >
-    <div
-      id="bcros-main-header__container"
-      class="flex flex-wrap content-center h-[68px] app-inner-container"
-    >
+    <div id="bcros-main-header__container">
```
Comment (patrickpeinanw) [inline src/components/bcros/header/Index.vue:7]: Is there a reason for removing these classes? It seems the header lost some vertical padding.

**Review by patrickpeinanw — APPROVED** [review]
Nice work making the tombstone responsive. I have some comments that might lead to some small tweak.

**Pull request merged**